src/platform-implementation-js/namespaces/user.ts

```diff
@@ -1,59 +1,64 @@
 import find from 'lodash/find';
-import * as ud from 'ud';
-import get from '../../common/get-or-fail';
 import type { Driver } from '../driver-interfaces/driver';
 import type { PiOpts } from '../platform-implementation';
-const memberMap = ud.defonce(module, () => new WeakMap()); // documented in src/docs/
```

Comment:
#910 touched on this a bit, but has something changed around using `ud` v. class members?

Reply:
I was uncomfortable with doing a blanket rip of ud, but I'm fine with it being taken out anywhere it gets in the way of something (like moving away from these weakmap standins for true private properties).

Reply:
I mimicked this [here](https://github.com/InboxSDK/InboxSDK/pull/1029/files#diff-36c20528977bbd53e3b13505ebf5a0c07cd1833807dbb66f70323df5b95e3bd0R15).